code/model/SiteTree.php

```diff
@@ -2643,19 +2609,18 @@ public function setMenuTitle($value) {
 	}
 	
 	/**
-	 * A flag provides the user with additional data about the current page status,
-	 * for example a "removed from draft" status. Each page can have more than one status flag.
-	 * Returns a map of a unique key to a (localized) title for the flag.
-	 * The unique key can be reused as a CSS class.
-	 * Use the 'updateStatusFlags' extension point to customize the flags.
+	 * A flag provides the user with additional data about the current page status, for example a "removed from draft"
+	 * status. Each page can have more than one status flag. Returns a map of a unique key to a (localized) title for
+	 * the flag. The unique key can be reused as a CSS class. Use the 'updateStatusFlags' extension point to customize
+	 * the flags.
 	 * 
 	 * Example (simple): 
-	 * "deletedonlive" => "Deleted"
+	 *   "deletedonlive" => "Deleted"
 	 * 
 	 * Example (with optional title attribute): 
-	 * "deletedonlive" => array('text' => "Deleted", 'title' => 'This page has been deleted')
+	 *   "deletedonlive" => array('text' => "Deleted", 'title' => 'This page has been deleted')
 	 *
-	 * @param Boolean $cached
+	 * @param bool $cached Whether to serve the fields from cache; false regenerate them
```

Comment:
@assertchris are you saying that passing `null` as a parameter to a function means it uses the default value? Because I've had that discussion before with another dev and tested the code and it's not the case.

Therefore, `getStatusFlags(null)` is not the same as `getStatusFlags()`

Here's a pretty rudementary proof ![](https://dl.dropboxusercontent.com/u/3429338/Clippings/2015-03/2015-03-04_23-31-35_pB5jD8pJfQ.png).

PS: sorry if I have the wrong end of the stick